voronoi_game.py

```diff
@@ -376,36 +377,37 @@ def check_move(self, move):
             return False
         is_valid = False
         if isiterable(move) and count_iterable(move) == 2:
-            if np.all([x is not None and sympy.simplify(x).is_real for x in move]):
+            if all(x is not None and not np.isnan(x) and np.isfinite(x) for x in move):
```

Comment:
Logic change: I don't know what simplifying and checking `is_real` does here, I assumed it was just a check that the move is composed of valid finite numbers